blog/_posts/2016-05-17-automated-data-validation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Automated Data Validation with Data Packages
+author: Dan Fowler
+username: dfowler
+projects: [dpm,frictionless-data,mira,goodtables-web]
+---
+
+Software projects have long benefited from
+[Continuous Integration](https://en.wikipedia.org/wiki/Continuous_integration)
+services like [Travis CI](http://travis-ci.org/),
+[Jenkins](https://jenkins.io), and others for ensuring and maintaining
+code quality over time.  On each commit into a project's shared
```

Comment:
Continuous integration is a process where on very update to a project's main repository all tests are automatically run and a report is generated. This allows developers to find and resolve errors quickly and reliably.

In addition, by displaying the "build status" others outside of the project can clearly see the status of the project test compliance.